src/modules/Convolution/info.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Convolution",
+    "description": "Image Convolution using a given 3x3 kernel matrix",
+    "inputs": {
+        "kernelValues": {
+            "type": "an array of 9 number values",
```

Comment:
I don't think we can take input type array, maybe we can do a string with nine space separated values.

Reply:
Okay I will do that! Thanks!